airflow/opt/providers/etna/etna/metrics/rollup_metrics.py

```diff
@@ -0,0 +1,58 @@
+import re
+from datetime import datetime
+from typing import Generator, Iterable, Tuple, Dict
+
+from airflow.models import DagBag, XCom, BaseOperator
+from prometheus_client import REGISTRY, Metric
+from prometheus_client.metrics_core import GaugeMetricFamily
+from prometheus_client.samples import Sample
+from pytz import utc
+
+
+rollup_label_metric_re = re.compile(r'label:(.*)')
+
+# magma_record_attributes[model_name, restricted, attribute_name, attribute_type] = count
+
+class Rollup:
+    def measure(self) -> Iterable[Tuple[Dict[str, str], int]]:
+        return []
+
+class RollupMetricsCollector:
+    def describe(self):
+        return []
+
+    def collect(self) -> Generator[Metric, None, None]:
+        bag = DagBag(read_dags_from_db=True)
+        bag.collect_dags_from_db()
+        for dag_name, dag in bag.dags.items():
```

Comment:
Collects all known dags and finds any that are labeled for rollup and generate RollupXcomOperator tasks.